### PR TITLE
Redirect route can be empty after registration

### PR DESCRIPTION
**Pull request by core23**
Rebase of #735

### Changelog

```markdown
### Fixed
- Fixed a potential empty route after calling `RegistrationFOSUser1Controller::registerAction`
```

### Subject
The route could be empty when no historic route was found in the user session.

### CODE REVIEW

**Comment by core23** [comment]
Please review @sonata-project/contributors

**Comment by OskarStark** [comment]
LGTM 👍

File: Controller/RegistrationFOSUser1Controller.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function registerAction()
                     $url = $this->get('session')->get('sonata_user_redirect_url');
```
Comment (greg0ire) [inline Controller/RegistrationFOSUser1Controller.php]: How about 
```
$url = $this->get('session')->get(
    'sonata_user_redirect_url', 
    $this->generateUrl('sonata_user_profile_show')
)
``` instead then ? I don't think `$url` can end up being null inside the if, only inside the else…

Comment (core23) [inline Controller/RegistrationFOSUser1Controller.php]: Fixed

**Pull request merged**